startup/10-devices.py

```diff
@@ -183,20 +183,25 @@ def _update_readback_setpoint(self):
 
     def put(self, value):
         self._setpoint = float(value)
-
+        
         start_time = ttime.monotonic()
         getattr(atf_db, f"put_{self._dtype}")(
             atf_db.get_channel_index(
                 f"{self._db}::{self._psname};{self._write_suffix}"
             ),
             value,
         )
-        while ttime.monotonic() - start_time < self._timeout:
-            if abs(self._get_readback() - value) < self._tol:
-                break
-            else:
-                # not reached yet, wait a bit
-                ttime.sleep(0.1)
+        try:
```

Comment:
I don't think we need `try`...`except` in this case as we do want it to raise an exception and stop the acquisition.